data-service/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "graphql-data-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.15.1",
+    "@typescript-eslint/parser": "^4.15.1",
+    "eslint": "^7.20.0",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-prettier": "^3.3.1",
+    "husky": "^4.3.8",
+    "prettier": "^2.2.1",
+    "prettier-eslint": "^12.0.0",
+    "pretty-quick": "^3.1.0",
+    "typescript": "^4.1.5"
+  },
+  "scripts": {
+    "lint": "eslint . --ext .ts --no-error-on-unmatched-pattern",
+    "pretty-quick": "pretty-quick",
+    "postinstall": "husky install"
+  },
+  "husky": {
```

Comment:
Please remove husky as discussed.

Reply:
Removed the Husky package. Using /config/hooks/pre-commit file instead